### PR TITLE
Added line break in accordion message.

### PR DESCRIPTION
**Pull request by brianseek**
## Description
Missed a line break in initial creation of accordions, this PR adds it in by splitting out second line as its own translation key and re-uses an existing key for the first sentence.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#40586

### CODE REVIEW

File: src/applications/check-in/locales/en/translation.json [hunk 1]
```diff
@@ -83,6 +82,7 @@
   "no": "No",
   "not-available": "Not available",
   "online-check-in-isnt-available-for-this-appointment-check-in-with-a-staff-member": "Online check-in isn’t available for this appointment. Check in with a staff member.",
+  "or-you-can-login-to-your-va-account-to-update-your-contact-information-online": "Or you can <0>{{link}}</0> to your VA account to update your contact information online.",
```
Comment (brdunfield) [inline src/applications/check-in/locales/en/translation.json:85]: I think we can also separate the spanish translation without needing to re-send to the state department

Comment (brianseek) [reply src/applications/check-in/locales/en/translation.json:85]: thanks for catching that, for some reason I was assuming that there wasn't a translation key for this in Spanish yet.

**Review by brdunfield — APPROVED** [review]
LGTM! Just verifying that 'sign in' is not translated yet, and so it's expected to see it in english on the spanish page?
I don't see it in my translation file so my assumption is that's the case!

👍

**Comment by brianseek** [comment]
> LGTM! Just verifying that 'sign in' is not translated yet, and so it's expected to see it in english on the spanish page? I don't see it in my translation file so my assumption is that's the case!
> 
> 👍

That is correct. It is in round 2 as part of that sentence though.

**Pull request merged**